src/metabase/core.clj

```diff
@@ -50,6 +61,7 @@
       mb-middleware/wrap-api-key         ; looks for a Metabase API Key on the request and assocs as :metabase-api-key
       mb-middleware/wrap-session-id      ; looks for a Metabase Session ID and assoc as :metabase-session-id
       mb-middleware/maybe-set-site-url   ; set the value of `site-url` if it hasn't been set yet
+      locale-negotiator                  ; Binds *locale* for i18n
```

Comment:
Does this just look at the `Accept-Language` header?

Reply:
Yes https://github.com/puppetlabs/clj-i18n/blob/master/src/puppetlabs/i18n/core.clj#L342